specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/examples/createWorkspace.json

```diff
@@ -64,6 +64,37 @@
           "description": "test description"
         }
       }
+    },
+    "202": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.MachineLearningServices/workspaces/testworkspace",
+        "name": "testworkspace",
+        "type": "Microsoft.MachineLearningServices/workspaces",
+        "location": "eastus2euap",
+        "sku": {
+          "name": "Basic",
+          "tier": "Basic"
+        },
+        "properties": {
+          "containerRegistry": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.ContainerRegistry/registries/testRegistry",
+          "keyVault": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.KeyVault/vaults/testkv",
+          "applicationInsights": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/microsoft.insights/components/testinsights",
+          "storageAccount": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/accountcrud-1234/providers/Microsoft.Storage/storageAccounts/testStorageAccount",
+          "discoveryUrl": "http://example.com",
+          "creationTime": "2017-03-01T23:14:37.0707808Z",
+          "friendlyName": "HelloName",
+          "description": "test description",
+          "encryption": {
+            "status": "Enabled",
+            "keyVaultProperties": {
+              "keyVaultArmId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/workspace-1234/providers/Microsoft.KeyVault/vaults/testkv",
+              "keyIdentifier": "https://testkv.vault.azure.net/keys/testkey/aabbccddee112233445566778899aabb",
+              "identityClientId": ""
+            }
+          },
+          "hbiWorkspace": true
```

Comment:
Well, the ModelValidation is reporting this field is illegal, because it is not in the definition of this struct. Is this field added in the [JEDI PR](https://github.com/Azure/azure-rest-api-specs/pull/7967)?

Reply:
Yes, both encryption and hbiWorkspace are added in the JEDI PR.